packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -78,7 +78,68 @@ export class AndroidEmulatorDevice extends DeviceBase {
     }, DISPOSE_TIMEOUT);
   }
 
+  private async shouldUpdateLocale(newLocale: Locale) {
+    const locale = newLocale.replace("_", "-");
+    const { stdout } = await exec(ADB_PATH, [
+      "-s",
+      this.serial!,
+      "shell",
+      "settings",
+      "get",
+      "system",
+      "system_locales",
+    ]);
+
+    // if user did not use the device before it might not have system_locales property
+    // as en-US is the default locale we assume that no value is the same as en-US
```

Comment:
I don't understand what does it mean "no value is the same as en-US"

Reply:
I modified the comment to explain it better